### DISCUSSION TITLE
GPT-4o's multimodal abilities?

### DISCUSSION THREAD

**Discussion by Henry-Suen** [lobehub/lobehub#2532] [General Question] [answered]
Hi, I'm not an experienced AI programmer. Right now, we can only use text generation through LobeChat. 
Is it because OpenAI's 4o API hasn't supported other inputs and outputs?
If it does in the future, are we gonna have video and voice interactions like they did in the official ChatGPT app on the mobile webpage?
I know it will be hard and change a lot so I'm not requesting it. Just wonder if there are plans on it.

**Comment by arvinxx** [comment, marked as answer]
> Is it because OpenAI's 4o API hasn't supported other inputs and outputs?

yes

>  are we gonna have video and voice interactions like they did in the official ChatGPT app on the mobile webpage?

yes

> Just wonder if there are plans on it.

yes
